analyzer/evmtokenbalances/evm_token_balances.go

```diff
@@ -294,6 +301,10 @@ func (m Main) Start() {
 	}
 }
 
+func (m *Main) cleanup() {
+	m.cfg.Source.Close()
```

Comment:
some Main's log error, some don't. this and evm_tokens being ones that don't

Reply:
:+1: added logging to those two